<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ServerPeer</class>
 <widget class="QMainWindow" name="ServerPeer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>537</width>
    <height>666</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>服务器端</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="gb_type">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>511</width>
      <height>311</height>
     </rect>
    </property>
    <property name="title">
     <string>配置</string>
    </property>
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>481</width>
       <height>41</height>
      </rect>
     </property>
     <property name="title">
      <string>类型</string>
     </property>
     <widget class="QRadioButton" name="rb_tcp">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>20</y>
        <width>51</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>TCP</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="rb_udp_one">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>20</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>UDP单播</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QRadioButton" name="rb_udp_group">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>20</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>UDP组播</string>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QRadioButton" name="rb_udp_broadcast">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>20</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>UDP广播</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>481</width>
       <height>80</height>
      </rect>
     </property>
     <property name="title">
      <string>UDP单播/广播</string>
     </property>
     <widget class="QPushButton" name="pb_remove">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>30</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>断开连接</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pb_init">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>30</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>连接</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_listen_port">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>50</y>
        <width>51</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>5001</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_client_ip">
      <property name="geometry">
       <rect>
        <x>65</x>
        <y>20</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>127.0.0.1</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>205</x>
        <y>50</y>
        <width>51</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>接收端口</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>9</x>
        <y>22</y>
        <width>51</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>客户端IP</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>193</x>
        <y>22</y>
        <width>60</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>客户端端口</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_client_port">
      <property name="geometry">
       <rect>
        <x>259</x>
        <y>20</y>
        <width>51</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>5002</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>180</y>
       <width>481</width>
       <height>51</height>
      </rect>
     </property>
     <property name="title">
      <string>UDP组播</string>
     </property>
     <widget class="QPushButton" name="pb_group_remove">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>20</y>
        <width>61</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>移除组播</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>205</x>
        <y>22</y>
        <width>51</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>组播端口</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pb_group_init">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>20</y>
        <width>61</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>加入组播</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_group_ip">
      <property name="geometry">
       <rect>
        <x>66</x>
        <y>20</y>
        <width>111</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>239.255.43.21</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>22</y>
        <width>51</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>组播地址</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_group_port">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>20</y>
        <width>51</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>5003</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>250</y>
       <width>481</width>
       <height>51</height>
      </rect>
     </property>
     <property name="title">
      <string>TCP</string>
     </property>
     <widget class="QPushButton" name="pb_tcp_stop">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>20</y>
        <width>61</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>停止监听</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>22</y>
        <width>51</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>监听地址</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pb_tcp_listen">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>20</y>
        <width>61</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>开始监听</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_tcp_listen_ip">
      <property name="geometry">
       <rect>
        <x>66</x>
        <y>20</y>
        <width>111</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>127.0.0.1</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>205</x>
        <y>22</y>
        <width>51</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>监听端口</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_tcp_listen_port">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>20</y>
        <width>51</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>5004</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QGroupBox" name="gb_receive">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>340</y>
      <width>511</width>
      <height>221</height>
     </rect>
    </property>
    <property name="title">
     <string>消息接收区</string>
    </property>
    <widget class="QTextBrowser" name="textBrowser">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>491</width>
       <height>192</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="gb_send">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>570</y>
      <width>511</width>
      <height>81</height>
     </rect>
    </property>
    <property name="title">
     <string>消息发送区</string>
    </property>
    <widget class="QTextEdit" name="te_send_info">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>441</width>
       <height>51</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pb_send">
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>20</y>
       <width>41</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>发送</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="netcomminicate.qrc"/>
 </resources>
 <connections/>
</ui>
